docs/supportedsites.md

```diff
@@ -1,1939 +1,1945 @@
-# Supported Sites
```

Comment:
Not entirely sure what happened here, I'd guess newlines `\n -> \n\r`, but please revert this.